### PR TITLE
update to use latest version of draftstore

### PR DESCRIPTION
**Pull request by robertparkinson**

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -46,6 +50,9 @@
     "idamOauth": "FEATURE_IDAM_OAUTH",
     "dashboard": "FEATURE_DASHBOARD"
   },
+  "documentManagement" : {
+  "url": "DOCUMENT_MANAGEMENT_API_GATEWAY_URL"
+},
```
Comment (rkondratowicz) [inline config/custom-environment-variables.json]: not related

File: config/default.json [hunk 2]
```diff
@@ -53,5 +57,8 @@
   "featureToggles": {
     "idamOauth": false,
     "dashboard": false
+  },
+  "documentManagement" : {
+    "url": "http://localhost:8085"
```
Comment (rkondratowicz) [inline config/default.json]: same as above

File: package.json [hunk 3]
```diff
@@ -26,7 +26,8 @@
   },
   "dependencies": {
     "@hmcts/cmc-common-frontend": "^1.17.1",
-    "@hmcts/draft-store-client": "^1.1.1",
+    "@hmcts/draft-store-client": "^1.2.1",
+    "@hmcts/cmc-draft-store-middleware": "^1.0.0",
```
Comment (timja-kainos) [inline package.json]: are you using this?

Comment (dharmendrak) [reply package.json]: used

**Pull request merged**